Assets/Scripts/Loot.cs

```diff
@@ -2,19 +2,78 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+[System.Serializable]
+public struct Loots {
+    public GameObject ressource;
+    public int number;
+}
+
 public class Loot : MonoBehaviour {
-    public GameObject[] loot;
-    private bool isQuitting;
+    public List<Item> tools;
+    public List<Loots> loots;
+    public int maxHealth;
+    public int stepNumber;
+
+    private int health;
+    private int step;
+    private int nextStep;
+    private int ressourcesByStep;
+
+    private void Start() {
+        health = maxHealth;
+        step = health / (stepNumber + 1);
+        nextStep = health - step;
 
-    private void OnApplicationQuit() {
-        isQuitting = true;
+        var totalRessources = 0;
+        foreach (var loot in loots) {
```

Comment:
you can write this more compact as `var totalRessources = loots.Sum(loot => loot.number);`